### PR TITLE
Enhancement. Reactive api rest for get SVG #216

### PR DESCRIPTION
**Pull request by davideduma**
Signed-off-by: davideduma <david24eduma@gmail.com>

**:heart:**

**What this PR does / why we need it**:  Reactive api rest for get SVG

**Special notes for reviewers**:

**Additional information (if needed):**

### CODE REVIEW

**Comment by tsurdilo** [comment]
@davideduma thanks for the pr. can you give some info on what it does?

File: api_rest/mvnw [hunk 1]
```diff
@@ -0,0 +1,316 @@
+#!/bin/sh
```
Comment (tsurdilo) [inline api_rest/mvnw:1]: do we need this?

**Comment by tsurdilo** [comment]
Think most of files are missing our copyright. please add

**Comment by davideduma** [comment]
> @davideduma thanks for the pr. can you give some more info on what it does?

With this functionality. It is possible to get the SVG string through a API REST

image

File: api_rest/mvnw (hunk 1 above)
Comment (davideduma) [reply api_rest/mvnw:1]: Good morning, yes that's right. the reason is beacause is a spring maven project.

**Comment by davideduma** [comment]
> Think most of files are missing our copyright. please add

Please give me a couple examples. i dont undestand

**Comment by tsurdilo** [comment]
Hi, you can look at any source files in the repo, for example: https://github.com/serverlessworkflow/sdk-java/blob/main/api/src/main/java/io/serverlessworkflow/api/deserializers/AuthDefinitionDeserializer.java#L1-L15

File: api_rest/src/main/java/io/serverlessworkflow/reactive_api_rest/ServerlessWorkFlowResponse.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder(toBuilder = true)
```
[the remark this answers is not in the document]
Comment (davideduma) [reply api_rest/src/main/java/io/serverlessworkflow/reactive_api_rest/ServerlessWorkFlowResponse.java]: this suggestion cannot be applied. The above line is required.

File: api_rest/src/main/java/io/serverlessworkflow/reactive_api_rest/RouterRest.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+import org.springframework.web.reactive.function.server.RouterFunction;
+import org.springframework.web.reactive.function.server.ServerResponse;
+
+@Configuration
+public class RouterRest implements IRouterRest {
+
+  /***
+   * Get image SVG from string.
+   * @param serverlessRequest
+   * @return
```
[the remark this answers is not in the document]
Comment (davideduma) [reply api_rest/src/main/java/io/serverlessworkflow/reactive_api_rest/RouterRest.java]: fixed issue

File: api_rest/src/main/java/io/serverlessworkflow/reactive_api_rest/RouterRest.java [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.reactive.function.server.RouterFunction;
+import org.springframework.web.reactive.function.server.ServerResponse;
+
+@Configuration
+public class RouterRest implements IRouterRest {
+
+  /***
+   * Get image SVG from string.
+   * @param serverlessRequest
```
[the remark this answers is not in the document]
Comment (davideduma) [reply api_rest/src/main/java/io/serverlessworkflow/reactive_api_rest/RouterRest.java]: Fixed issue

**Comment by davideduma** [comment]
this suggestion cannot be applied. The above line is required.:

image

**Comment by davideduma** [comment]
Fixed issues sonatype

**Comment by davideduma** [comment]
Hi. Can you help me. This pull request still in progress. Thanks!

**Comment by davideduma** [comment]
> Hi. Can you help me. This pull request still in progress. Thanks!

Only Awaiting approval to owner.

**Review by tsurdilo — APPROVED** [review]

**Pull request merged**